changelog.json

```diff
@@ -1,5 +1,20 @@
 {
   "releases": {
+    "1.6.3": [
+      "[Added] Provide option to discard when files would be overwritten by a checkout - #6755. Thanks @mathieudutour!",
+      "[Fixed] Context menu doesn't open when right clicking on the edges of files in Changes list - #6296. Thanks @JQuinnie!",
+      "[Fixed] Display question mark image when there is no commit selected in dark theme - #6915. Thanks @say25!",
```

Comment:
```suggestion
      "[Fixed] Display \"question mark\" image when no commit selected in dark theme - #6915. Thanks @say25!",
```

Reply:
Change image to icon.

Reply:
@iAmWillShepherd I was gonna change it but I ended up reverting it because the icon is a part of the image so that still feels more fitting...